### DISCUSSION TITLE
Issues connecting to Supabase recently

### DISCUSSION THREAD

**Discussion by jasonho-lynx** [supabase/supabase#1541] [Questions] [answered]
Hi there,

Not sure if I'm imagining things, but since last week I've been having more issues connecting to the Supabase server, and wanted to check if anyone else is facing the same.

For example, when after I login to app.supabase.io, the 'Table Editor' page will be stuck loading. If I click on 'Database', occasionally I get an error about unexpected JSON token at position 0. After refreshing a few times, both these pages will eventually work.

Sometimes, the same happens when using `supabase-js`. If I do something like `await supabase.from(<TABLE>).delete()`, an error message along the lines of `Database client error... Error connecting to database` will appear. Again, after trying it a couple of times, it will work fine. EDIT: a new error message showed up today - 

```
{"details":"server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n","code":"","message":"Database connection error. Retrying the connection."}
```

Unfortunately, because I can't reproduce these reliably, I don't have the actual error messages right now. Will update when I do. Just wanted to check if it's something that others are also experiencing, or whether it sounds like something I messed up on my end.

Thanks!

**Comment by inian** [comment, marked as answer]
Can you email your project id to beta@supabase.io @jasonho-lynx? Will help check on the project

**Reply by jasonho-lynx** [reply]
I've sent it, thank you for the quick reply @inian!

**Reply by jasonho-lynx** [reply]
Confirmed over email that this was because my database instance ran out of memory, likely due to some rogue query. Will need to find and fix that issue. 

@inian reset the instance for me and it works now. Thanks again!
